demo.js

```diff
@@ -88,11 +88,25 @@ function createGrid() {
 		}
 		var cls = cls.join(' ').trim();
 
-		$grid.append('<div' + (cls ? ' class="' + cls + '"' : '') + '>' + (encodeURIComponent(cell.text || index)) + '</div>');
-		htmlText += "    " + '<div' + (cls ? ' class="' + cls + '"' : '') + '>' + (encodeURIComponent(cell.text || index)) + '</div>' + "\r\n";
+		$grid.append('<div' + (cls ? ' class="' + cls + '"' : '') + '>' + format(encodeURIComponent(cell.text || index)) + '</div>');
+		htmlText += "    " + '<div' + (cls ? ' class="' + cls + '"' : '') + '>' + format(encodeURIComponent(cell.text || index)) + '</div>' + "\r\n";
```

Comment:
Why do we need to use encodeURIComponent() here? It converts simple text like "Hello Friends" to "Hello%20Friends" while setting cell's content.

Reply:
We need it in order to prevent injections of malicious code. Like <script>...</script>.